domain/induction/src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/induction/dto/CreateInPrisonInterestsDto.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.dto
+
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.InPrisonTrainingInterest
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.InPrisonWorkInterest
+
+data class CreateInPrisonInterestsDto(
+  val inPrisonWorkInterests: List<InPrisonWorkInterest>,
```

Comment:
I was about to question about a DTO that has child members that are the domain objects (and slightly ridicule / compare you with someone else we know! 🤣 ) .... but these are enums? Yes? `InPrisonWorkInterest` is a domain enum, not a domain class?

Reply:
A few of the classes are domain classes actually, but they are simple value objects with no identity, so I think it's ok.